examples/foundationdb/README.md

```diff
@@ -58,6 +74,10 @@ cluster.
 
 Please follow below command to create the CR, please make sure to use the local_cluster.yaml file from this repo.
 
+**NOTE:**
+
+Replace `<registry>`, `<account_name>` and `<tag_name>` for the `imageName` value in `./local_cluster.yaml` before running following command.
```

Comment:
```suggestion
Replace `<registry>`, `<repository>` and `<tag_name>` for the `imageName` value in `./local_cluster.yaml` before running the following command.
```

Reply:
I don't think we should change it to repository, actually it is the username that is prepended with repository (foundationdb).
So it would be `account_name/foundationdb`.

Reply:
@akankshakumari393 in the command documented above, we are using `docker push <registry>/<repository>/foundationdb:<tag_name>`. We should use the same terms at other places